docs/deployments/oc10-app.md

```diff
@@ -9,40 +9,36 @@ geekdocFilePath: oc10-app.md
 
 {{< toc >}}
 
-One of the build artifacts of the ownCloud Web is a ownCloud 10 app bundle.
+The ownCloud Web is being deployed as an app to [ownCloud marketplace](https://market.owncloud.com) to enable easy early integration into existing ownCloud 10 instances.
+After completing this setup, ownCloud Web will be available on `https://<your-owncloud-server>/apps/web`.
+
+{{< hint info >}}
+Depending on your setup, it might be required to use `apps-custom` instead of `apps` in the URL. Keep this in mind when setting all mandatory URL addresses.
```

Comment:
In new oC10 installs, in the default `config.php`, `apps-paths` has `apps-external` which is where "extra" apps are installed. Will installations like this have to put `apps-external` in the URL?

If the URL is based on the actual folder name which the app is installed in, then should we use `apps-external` here?

Or is there some way that we can make the URL always be `/apps/web` ?

Reply:
I don't have much knowledge about how and where marketplace apps are installed. But I guess it will always land in the external folder - always have to be `external/web`?

We saw `custom` folder in the setup of @micbar Maybe that's some custom change though? I guess we can adjust this to mention `apps-external` and warn that it could have even different name?

Reply:
yes. use `apps-external`

Reply:
Done :wink: